qt/scientific_interfaces/ISISReflectometry/test/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  add_compile_options(-Wno-maybe-uninitialized)
+endif()
+
```

Comment:
I see this is repeated in a lot of CMakeLists, is it possible for this to go in a more common location?

Perhaps in here https://github.com/mantidproject/mantid/blob/main/buildconfig/CMake/CommonSetup.cmake or somewhere similar?

Reply:
Or perhaps this is the location for it? https://github.com/mantidproject/mantid/blob/fc864399055dc8c96200a84c1c521da3f96668aa/buildconfig/CMake/GNUSetup.cmake#L101

Reply:
I could do that but there would be a list of test folders specified in e.g. `GNUSetup.cmake`, so the build configuration for those test folders would then be split between their local `CMakeLists.txt` and the other file.

Reply:
Oh I see,  so we only want this compile option for these test libraries?

Reply:
In that case ignore me! Looks fine as it is :)

Reply:
Haha thanks, yes it's only some specific test folders that need the compiler flag.